src/matlab2tikz.m

```diff
@@ -1519,33 +1539,29 @@ case guitypes()
     % a reasonable default.
     matlabDefaultNumMinorTicks = 3;
     if ~isempty(ticks)
-        options = opts_add(options, ...
-            [axis,'tick'], sprintf('{%s}', ticks));
+        options = opts_add(options, [axis,'tick'], sprintf('{%s}', ticks));
```

Comment:
I agree that the added beauty is minimal. But I always had the same impression as you had: "why is that newline there?". During working on this PR, I scrolled over this part, it bothered me a few times too often, so I changed it to avoid it bothering me in the future.

Personally, I think it still makes sense to limit the width of a line: the longer a line becomes, the harder it becomes to keep track of each different line. Indirectly, it limits the indentation depth within a function (i.e. complexity). 

On modern wide screens, you indeed have all the width you could wish, but limiting that makes that you can have 2 windows next to each other.
Alternatively, you can turn such a screen and get 80 chars per line and about 80 lines as you can see below:
![screen shot 2015-11-30 at 16 25 24](https://cloud.githubusercontent.com/assets/177360/11475729/1d5f940a-977f-11e5-8c71-f3483ff02888.png)


Personally, I avoid going over 80 chars unless readability of the alternative is worse.